docs/website/contents/for-developers/CivicTime.md

```diff
@@ -0,0 +1,227 @@
+In particular, until the oldest such block is after the voting deadline, a rollback could switch to a chain with a different voting outcome in that epoch.
+For example, switching from a chain in which the next era is only a few slots away to one in which it is at least another epoch away.
+
+*Aside*.
+On the other hand, when using a (non-recent) ledger state in a historical era, the HFC could theoretically use its knowledge about the upcoming eras in order to translate even further ahead.
+For example, the HFC could safely assume the one second duration for slots between some given Babbage ledger state and however many slots it might possibly take before that ledger state could transition from Babbage to Conway and from Conway to whatever comes after, because that mystery era is the first time the slot duration could change.
+The HFC does not do this in practice, since the extra complexity is not worthwhile; it's not necessary for syncing nodes to be smarter than caught-up nodes.
+
+[^EmergencyRapidity]: Technicality.
+    It could perhaps be weeks or days in an emergency, but a change to the slot duration would almost certainly be avoided in that case.
+
+## What About Chain Growth Failures?
+
+At a high-level, today's HFC was derived as follows.
+The exact thought process was not recorded during the design work, but this is a plausible reconstruction.
+
+  - {ImmutableTranslations}.
+    Because of humans' general assumptions about time, it would be prohibitively confusing if the node (even its internal interfaces) might give different translations of the same slots/civic times.
+    So a switch from `t1` to `t2` for the result of some translation is unacceptable.
+    At the very least, the node should refuse to do a translation that might be invalidated by on-chain governance in the meantime (eg several weeks into the future).
+
+  - {NonemptyForecastRange}.
+    On the other hand, the node should be able to translate slot/civic times slightly ahead of its tip.
+    At the very least, some users presumably want some capacity to plan ahead (eg to smartly schedule their node's brief downtime).
+
+  - {RollbackInsensitiveTranslations}.
+    As a specific subcase of the first requirement, the node should even refuse to do translations that might be invalidated by a rollback.
+
+  - {MonotonicTranslations}.
+    It would also be too confusing if a node agreed to translate some slot/civic time and then subsequently refused to translate that same slot/civic time.
+    So `Just t1 -> Nothing` is also unacceptable for some translation, not merely `Just t1 -> Just t2`.
+
+  - {DespiteChainGrowthViolation}.
+    All of the above must hold even in the presence of a Chain Growth violation, except switching from a chain that violates Chain Growth to one that does not is allowed to violate the RollbackInsensitiveTranslations subcase of ImmutableTranslations.
```

Comment:
It seems even less likely to me that *this* was a design goal as opposed to "chain growth violation tolerance in all circumstances" which is discussed in https://github.com/IntersectMBO/ouroboros-consensus/pull/1320/#discussion_r1856324546

Reply:
Yeah, agreed. I lost sight of that context at the top of the section—myopia just trying to get accurate wording in the details.

I'll separate out this last bullet point as a "retroactive justification"/unintended-but-defensible behavior of the current design tomorrow.

Reply:
Actually, does the clarifying footnote I just added seem to suffice in your opinion?

Reply:
Sounds good :+1: